homeassistant/components/melcloud/strings.json

```diff
@@ -50,5 +50,27 @@
       "title": "The MELCloud YAML configuration import failed",
       "description": "Configuring MELCloud using YAML is being removed but there was an connection error importing your YAML configuration.\n\nEnsure connection to MELCloud works and restart Home Assistant to try again or remove the MELCloud YAML configuration from your configuration.yaml file and continue to [set up the integration](/config/integrations/dashboard/add?domain=melcoud) manually."
     }
+  },
+  "entity": {
+    "sensor": {
+      "room_temperature": {
+        "name": "Room Temperature"
```

Comment:
Oh with that suggestion I meant, use sentence case instead of title case (or however you call it)

Reply:
Oh sorry, didn't understand. I will fix...

Reply:
No problem :)